yt_dlp/extractor/itv.py

```diff
@@ -113,12 +113,17 @@ def _call_api(self, video_id, playlist_url, headers, platform_tag, featureset, f
 
     def _get_subtitles(self, video_id, variants, ios_playlist_url, headers, *args, **kwargs):
         subtitles = {}
-        platform_tag_subs, featureset_subs = next(
-            ((platform_tag, featureset)
-             for platform_tag, featuresets in variants.items() for featureset in featuresets
-             if try_get(featureset, lambda x: x[2]) == 'outband-webvtt'),
-            (None, None))
-        if platform_tag_subs or featureset_subs:
```

Comment:
why not just
```diff
         platform_tag_subs, featureset_subs = next(
             ((platform_tag, featureset)
-             for platform_tag, featuresets in variants.items() for featureset in featuresets
+             for platform_tag, featuresets in reversed(variants.items()) for featureset in featuresets
              if try_get(featureset, lambda x: x[2]) == 'outband-webvtt'),
             (None, None))
-        if platform_tag_subs or featureset_subs:
+        if platform_tag_subs and featureset_subs:
```
It is simpler and the intend to use the last item is clear from `reversed`

Reply:
yeah, sure